plvp.config

```diff
@@ -5,7 +5,9 @@ local:
     pprofaddr: ':55557'
     startscamp: true
     rootca: /root.crt
+    interface: en0
```

Comment:
I don't know this code at all, but should this be `net1` by default, or is `en0` a better default for most situations?